server/src/pages/leave-screenshots/view.js

```diff
@@ -24,18 +24,16 @@ class Body extends React.Component {
     }
     return (
       <reactruntime.BodyTemplate {...this.props}>
-        <div className="column-center full-height inverse-color-scheme">
-          <div className="large-icon-message-container">
-            <div className="large-icon logo" />
-            <div className="large-icon-message-string">
-              This will permanently erase all of your Firefox Screenshots data.
-            </div>
-            <form action="/leave-screenshots/leave" method="POST" className="responsive-wrapper row-center">
+        <div className="column-center full-height alt-color-scheme">
+          <img src="/static/img/image-nope_screenshots.svg" alt="no Shots found" width="432" height="432"/>
```

Comment:
Should this be `src="/static/img/..."` or should we use that `src={this.props.staticLink("/static/img/...")}` like we do above for the `<link>` tag?